cristin-biobank/src/main/java/no/unit/nva/biobank/model/QueryBiobank.java

```diff
@@ -0,0 +1,36 @@
+package no.unit.nva.biobank.model;
+
+import static no.unit.nva.biobank.model.ParameterKeyBiobank.IGNORE_PATH_PARAMETER_INDEX;
+import static no.unit.nva.biobank.model.ParameterKeyBiobank.PATH_BIOBANK;
+import static no.unit.nva.biobank.model.ParameterKeyBiobank.PATH_IDENTITY;
+import java.util.Map.Entry;
+import no.unit.nva.cristin.model.CristinQuery;
+
+public class QueryBiobank extends CristinQuery<ParameterKeyBiobank> {
+
+    public static  QueryBuilderBiobank builder() {
+        return new QueryBuilderBiobank();
+    }
+
+    @Override
+    protected String getNvaPathItem(int pathSize, Entry<ParameterKeyBiobank, String> entry) {
+        return entry.getKey().equals(PATH_IDENTITY)
+                   ? PATH_BIOBANK.getNvaKey()
+                   : entry.getKey().getNvaKey();
+    }
+
+    @Override
+    protected boolean ignoreNvaPathParameters(Entry<ParameterKeyBiobank, String> entry) {
+        return entry.getKey().ordinal() > IGNORE_PATH_PARAMETER_INDEX;
```

Comment:
`ignorePathParameters `do the same as this method, is that intended?

Reply:
That is a really good question.

The code was much more complicated when I first made it.
It can be simplified

Reply:
But that should be in a refactoring.